docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -0,0 +1,127 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ledger-api-test-tool:
+
+Ledger API Test Tool
+####################
+
+The Ledger API Test Tool is a command line tool for testing the correctness of
```

Comment:
What does "test the correctness of" mean? A concrete example would make this clearer - something like "For example, it will show you if there is x problem with your implementation"